### PR TITLE
IMP file.task: switch active to enabled

### PR DESCRIPTION
**Pull request by bealdav**

### CODE REVIEW

File: external_file_location/models/task.py [hunk 1]
```diff
@@ -138,6 +138,7 @@ def _template_render(self, template, record):
     def run_task_scheduler(self, domain=None):
         if domain is None:
             domain = []
+        domain.append([('enabledd', '=', True)])
```
Comment (bguillot) [inline external_file_location/models/task.py]: There is a typo s/enabledd/enabled

Comment (bguillot) [inline external_file_location/models/task.py]: Are you sure the domain is correct ?
For me there is too many levels.

It should be  : domain.append(('enabled', '=', True)) no ?

**Review by bguillot — CHANGES_REQUESTED** [review]

File: external_file_location/models/task.py (hunk 1 above)
Comment (bealdav) [reply external_file_location/models/task.py]: oula bienvu

Comment (bealdav) [reply external_file_location/models/task.py]: merci

**Comment by bealdav** [comment]
@bguillot now fixed

**Review by bguillot — APPROVED** [review]
Merci !

**Pull request merged**